gui/wxpython/modules/import_export.py

```diff
@@ -407,6 +408,7 @@ def __init__(self, parent, giface, link=False):
         self.link = link
 
         self.layersData = []
+        self.showModulesDisplayInfo = Signal('GdalImportDialog.showModulesDisplayInfo')
```

Comment:
This signal name should primarily express what happened. This class does not know and does not need to know what happens when the signal is emitted. Name like `importSuccessful` is a better fit.

Reply:
renamed to showImportSuccessfulInfo